lib/src/sai_redis_route.cpp

```diff
@@ -376,8 +376,8 @@ const sai_route_api_t redis_route_api = {
 
     REDIS_GENERIC_QUAD_API(route_entry)
 
-    sai_bulk_create_route_entry,
-    sai_bulk_remove_route_entry,
-    sai_bulk_set_route_entry_attribute,
-    sai_bulk_get_route_entry_attribute,
+    redis_bulk_create_route_entry,
```

Comment:
Do you think it would be worth creating some sort of macro like REDIS_GENERIC_BULK_QUAD_API or something similar like we have for the normal APIs?

Reply:
yes, something similar like this: https://github.com/Azure/sonic-sairedis/blob/master/lib/inc/sai_redis_internal.h#L74 but for bulk, the issue is that validation part of entry needs to be moved sai_meta.cpp like single object validation part, and a extra method for serialization must be selected, and this is a big chang, but at some point i will make this when i will be refactoring meta validation part

Reply:
Sounds good!